source/fundamentals/linq.txt

```diff
@@ -1,8 +1,8 @@
 .. _csharp-linq:
 
-====
-LINQ
-====
+==========================================
+Query with LINQ and Aggregation Operations 
```

Comment:
This title makes it sound like the page covers two ways to query: LINQ and aggregation operations. LINQ is really just an alternative syntax for running queries--one which happens to use aggregation operations under the hood. Is something like 'Query Data with LINQ' too short?

Reply:
Yes it's too short unfortunately :(